server/controllers/reviewsController.js

```diff
@@ -0,0 +1,43 @@
+import {
+  listReviews,
+  addReview,
+  updateHelpfulness,
+  markAsReported,
+}
+  from '../model/reviewsModel.js';
+
+export const getReviews = (req, res) => {
+  listReviews(req.query, (err, results) => {
+    if (err) {
+      console.log('error getting reviews...', err);
```

Comment:
You should also send back a response to the client in the case of an error. You can use something like
`res.status(500).send('error getting reviews...')`